dumper/app/src/test/java/com/google/edwmigration/dumper/application/dumper/connector/teradata/MetadataQueryGeneratorTest.java

```diff
@@ -144,30 +101,6 @@ public void createSelectForDatabasesV_usersAndDbsLimit() {
         query);
   }
 
-  @Test
```

Comment:
Can we have a test for DBKind as I still see that filtering is available?

Reply:
The `DBKind` column filtering is tested above in the `createSelectForDatabasesV_usersAndDbsLimit` test. Do you have any other test in mind?

Reply:
ah sorry, missed the collapsing bit. The test above is good enough